Aliucord/src/main/java/com/aliucord/api/SettingsAPI.java

```diff
@@ -203,4 +203,33 @@ public void setString(String key, String val) {
     public void setObject(String key, Object val) {
         settings.setObject(key, val);
     }
+
+    /**
+     * Get a value of an unknown type
+     * @param key Key of the item
+     */
+    public Object getUnknown(String key, Object defValue) {
+        Object value = null;
+        if (defValue instanceof String) value = getString(key, (String) defValue);
+        else if (defValue instanceof Boolean) value = getBool(key, (Boolean) defValue);
+        else if (defValue instanceof Long) value = getLong(key, (Long) defValue);
+        else if (defValue instanceof Float) value = getFloat(key, (Float) defValue);
+        else if (defValue instanceof Integer) value = getInt(key, (Integer) defValue);
+        else value = getObject(key, defValue);
+        return value;
```

Comment:
get rid of the value variable and instead return right away, then you can also drop all the else